### PR TITLE
Add users and comments

### PR DESCRIPTION
**Pull request by fatfroggo**

### CODE REVIEW

**Comment by fatfroggo** [comment]
also adds deleting comment functionality (i forgot to switch branches - sorry!)

File: src/components/CommentAdder.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const CommentAdder = ({review_id, setComments}) => {
+
+const {user} = useContext(UserContext)
+
+const [newComment, setNewComment] = useState("");
+
+const handleSubmit = (e) => {
+    e.preventDefault()
+    setComments((currentComments) => {
+        console.log(newComment)
```
Comment (yalicee) [inline src/components/CommentAdder.jsx]: get rid of console.log

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,5 +1,17 @@
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
+
+
 const Header = () => {
-    return <h1>NC Games</h1>
+
+    const {user} = useContext(UserContext)
+    console.log(user)
```
Comment (yalicee) [inline src/components/Header.jsx]: remove

File: src/components/CommentAdder.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+return (
+<form className="comment-adder" onSubmit={handleSubmit}>
+  <label htmlFor="newComment">Add a comment</label>
+  <input
+    id="newComment"
+    value={newComment}
+    onChange={(e) => setNewComment(e.target.value)}
+    required
+  ></input>
```
Comment (yalicee) [inline src/components/CommentAdder.jsx]: change this to text area

**Review by yalicee — APPROVED** [review]
Looks really good Lily. Approved :)  Check comments before merge though

**Pull request merged**